libraries/math/src/instruction.rs

```diff
@@ -159,7 +182,30 @@ pub fn f32_divide(dividend: f32, divisor: f32) -> Instruction {
     }
 }
 
-/// Create PreciseSquareRoot instruction
+
+/// Create F32 Multiplication instruction
```

Comment:
Hehe, looks like the old ones got fixed, but not the new ones
```suggestion
/// Create F32 Exponentiate instruction
```